app/views/home/index.html.erb

```diff
@@ -1,109 +1,36 @@
 <div class="govuk-grid-row">
-  <h2 class="govuk-heading-l govuk-grid-column-full">Overview</h2>
-  <div class="govuk-grid-column-full">
-    <div class="govuk-grid-column-one-third">
-      <div class="govuk-panel govuk-panel--confirmation dashboard-colour">
+  <div class="govuk-grid-column-full" id="overview">
+    <h2 class="govuk-heading-l govuk-grid-column-full">Overview</h2>
+    <div class="govuk-grid-column-one-third" id="team-members">
+      <div class="govuk-panel govuk-panel--confirmation summary-background">
         <h1 class="govuk-panel__title text-left">
           <%= @team_members.count %>
         </h1>
         <p class="govuk-panel__body text-left govuk-!-font-size-27">
-          Team Members
+          <%= link_to "#{"Team Member".pluralize(@team_members.count)}", team_members_path, class: "govuk-link govuk-link--no-visited-state summary-link-colour" %>
```

Comment:
I'd be happy pluralizing these in all cases - when it's a status screen like this I think the plural headings ("Team Members") make sense even if it's 0 or 1.

c.f. [Locations](https://github.com/alphagov/govwifi-admin/pull/392/files#diff-d9db7f7cb7dcd9bbe759b990b685bddaR20) and [IPs](https://github.com/alphagov/govwifi-admin/pull/392/files#diff-d9db7f7cb7dcd9bbe759b990b685bddaR30).

(Sorry, missed this first time round! Not a big deal, could be a "merge and deal with later" situation)

Reply:
_Status Screen_. Never thought about it that way. 🤔 
Now that I think about it, leaving them as plurals does look better and consistent. Thanks!